perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -735,25 +735,55 @@ sub admin_stats {
                     'latitude', 'longitude',
                     'cobrand',  'category',
                     'state',    'user_id',
-                    'external_body'
-                ]
+                    'external_body',
+                    'title', 'detail',
+                    'photo',
+                    'confirmed',
+                    'whensent',
+                    'service',
+                    'extra',
+                ],
             }
         );
-        my $body = "ID,Created,E,N,Category,Status,UserID,External Body\n";
+        my $body = "Report ID,Created,Sent to Agency,Last Updated,E,N,Category,Status,UserID,External Body,Title,Detail,Media URL,Interface Used,Council Response\n";
+        require Text::CSV;
+        my $csv = Text::CSV->new({ binary => 1 });
         while ( my $report = $problems->next ) {
             my $external_body;
             my $body_name = "";
             if ( $external_body = $report->body($c) ) {
-                $body_name = $external_body->name;
+                $body_name = eval { $external_body->name } || '[Unknown body]';
             }
-            $body .= join( ',',
-                $report->id,           $report->created,
+            my $detail = $report->detail;
+
+            # replace newlines with slash
```

Comment:
What's the reasoning behind this? Can the CSV exporter not handle newlines in fields?